compiler/damlc/tests/daml-test-files/InterfaceViewNonSerializable.daml

```diff
@@ -2,7 +2,7 @@
 -- SPDX-License-Identifier: Apache-2.0
 
 -- @SINCE-LF-FEATURE DAML_INTERFACE
--- @ERROR Interface I has a view method which returns a non-serializable type InterfaceViewNonSerializable:NonSerializable
+-- @ERROR expected serializable type
```

Comment:
I agree with the simplification but I would also like the error message to be as concrete as possible. If I see `expected serializable type` with no other context I'm not sure what needs to be fixed.
If the error does contain the context information from `ContextDefInterface` (as I hope it does), could you please update this line to show the context too?